### ISSUE TITLE
Bug: Can't Launch Military Campaigns.

### ISSUE THREAD

**Issue by Jessen21** [pmotschmann/Evolve#663]
as the title says, when I click on the attack button, nothing happens. Regardless of how many soldiers I devote to it.

**Comment by Jessen21** [comment]
I am not sure what was doing it. I have done a reset since, and can't reproduce it.

**Comment by Beorseder** [comment]
Can you do Restore Backup in Settings to load in the save from the previous run and then paste it here?
Make sure you export your current save first though and put it somewhere so you can import it back in afterwards.

**Comment by pmotschmann** [comment]
If you have a save file from when it wasn't working, that would help tremendously to isolate any potential issue.

**Comment by Jessen21** [comment]
I tried to do a backup, its reloading my current run so I'm not able to get back to the previous.

**Comment by NilanthAnimosus** [comment]
I have strangely encountered this bug and I somehow fixed it same-run. I started having this issue around the time I had gotten MAD research done in terms of progression (Very early space) - And I fixed it just after bioseed ship construction.

In checking for reproducibility by importing my export of the bugged run I ended up fixing the issue in my game. sadly if my exported data fixed itself then it may be an issue that either corrects itself later or does not reproduce with an export (Or has conditions to reproduce via exporting).

I did try page and browser refreshes which did not fix it, but somehow reimporting an export did - Which I strongly feel I would have tried days ago but no longer recall sadly, only certain that it fixed it once I had built a bioship (But not reset).

What I can say I noticed before hitting and loading my export is that the relations value decreases when you click attack, no log entry was shown for the attack and your troop counts do not decrease nor injured count like the attack does not occur alongside not appearing to gain resources. before re-importing I had checked my message log settings message counts (all were the default as I never changed it) and combat was shown and the tab was on all.
